python/cugraph/ktruss/ktruss_subgraph_wrapper.pyx

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+from cugraph.ktruss.ktruss_subgraph cimport *
+from cugraph.structure.c_graph cimport *
+from cugraph.utilities.column_utils cimport *
+from libcpp cimport bool
+from libc.stdint cimport uintptr_t
+from libc.stdlib cimport calloc, malloc, free
+from libc.float cimport FLT_MAX_EXP
+
+import cudf
+import cudf._lib as libcudf
+import rmm
+import numpy as np
+
+
+def ktruss_subgraph(graph_ptr,k, subgraph_truss):
+    """
+    Call gdf_katz_centrality
+    """
+    cdef uintptr_t graph = graph_ptr
+    cdef Graph* g = <Graph*>graph
+
+	# cdef int kc = <int>k
+    cdef uintptr_t subgraph = subgraph_truss
+    cdef Graph* gsub = <Graph*>subgraph
```

Comment:
cdef uintptr_t subgraph = graph_wrapper.allocate_cpp_graph()
    cdef Graph* gsub = <Graph*>subgraph

Reply:
subgraph is still a WIP and will be addressed in a future PR.